File: Modul6/Python/PRAK601-2310817210003-MuhammadAzrianzan.py
baris, kolom = input("").split()
baris = int(baris)
kolom = int(kolom)
index = 0
myList = list(map(int, input().split()))

for i in range(baris):
    for j in range(kolom):
        print(myList[index], end=" ")
        index = index + 1
    print()

File: Modul6/C/PRAK601-2310817210003-MuhammadAzrianzan.c
#include <stdio.h>

int main() {
    int baris, kolom;
    int index = 0;
    scanf("%d %d", &baris, &kolom);
    int ukuran = baris * kolom;
    int myarr[ukuran];
    for (int j = 0; j < ukuran; j++) {
        scanf("%d", &myarr[j]);
    }

    for (int i = 0; i < baris; i++) {
        for (int x = 0; x < kolom; x++) {
            printf("%d ", myarr[index]);
            index++;
        }
        printf("\n");
    }
        
    return 0;
}